src/components/formProducts/FormProducts.js

```diff
@@ -99,19 +97,40 @@ const FormProducts = ({ items }) => {
         </div>
         <div className="productos_view products-container">
           {itemsFiltered?.length > 0 ? (
-            itemsFiltered.map((item, index) => (
-              <ProductForList
-                key={`${index}-${item.name}`}
-                item={item}
-                token={token.current}
+            <>
+              <ProductsByEstimation
+                items={itemsFiltered}
+                start={0}
+                end={7}
+                title="Soon"
+                estimationType="soon"
               />
-            ))
+              <ProductsByEstimation
+                items={itemsFiltered}
+                start={8}
+                end={15}
```

Comment:
Valores de las propiedades `start` and `end` **no coincide**n con el AC definido en el [issue 12](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/issues/12).

>  Necesita comprarlo más o menos pronto (entre 7 y 30 días)

![image](https://user-images.githubusercontent.com/8689897/158035454-912daeff-7725-4a9b-9f18-84a1ba2b47e2.png)

Por favor, te agradecería su cambio:

```suggestion
                start={7}
                end={30}
```